static/sass/styles.scss

```diff
@@ -284,3 +284,10 @@ summary {
     }
   }
 }
+
+// Advantage section
+#enterprise-table { // sass-lint:disable-line no-ids
```

Comment:
If we have control over this element, why add an id and not a class?

Reply:
Good point